PulsePlayer/PulsePlayer/ios/NextAdThumbnailController.h

```diff
@@ -0,0 +1,27 @@
+//
+//  NextAdThumbnailController.h
+//  PulsePlayer
+//
+//  Created by Sravani Kancharla on 2021-03-09.
+//  Copyright © 2021 Ooyala. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+@protocol NextAdThumbnailControllerDelegate <NSObject>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@end
+
+@interface NextAdThumbnailController : UIViewController
+
+@property (weak, nonatomic) id<NextAdThumbnailControllerDelegate> delegate;
+@property (strong, nonatomic) IBOutlet UIView *NextAdView;
+@property (strong, nonatomic) IBOutlet UIImageView *NextAdImageView;
```

Comment:
I see NextAdImageView being used only in extension class i.e. "NextAdThumbnailController.m" file. If it is not in use anywhere else, I would have preferred to declare it in extension class itself. Also, change it to *nextAdImageView i.e. first letter starting with small letter.